pyworkflow/__init__.py

```diff
@@ -179,6 +179,17 @@ def getPythonLibFolder():
         from sysconfig import get_paths
         return join(get_paths()['data'], "lib")
 
+    @staticmethod
+    def debugOn(*args):
+        from pyworkflow.utils import envVarOn
```

Comment:
We should avoid imports out of the top unless very weird cases.

Reply:
Yep, this is the case to avoid import errors. we are reusing (we can may be avoid it) envVarOn from utils, and utils uses Config from this file in the Logger... We should probably refactor all that but I guess this is not the time for it.

Reply:
Ok, let's move on

Reply:
Ok, let's move on